### PR TITLE
NR-83107 - provision alerts with terraform

### PR DESCRIPTION
**Pull request by rubenruizdegauna**

### CODE REVIEW

File: .github/workflows/component_canary_alerts_delete.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+name: .. 🚧 Canary alerts | Create
```
Comment (brushknight) [inline .github/workflows/component_canary_alerts_delete.yml:1]: typo

File: .github/workflows/component_canary_alerts_delete.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ env.AWS_ASSUME_ROLE }}
+          aws-region: us-east-2
+
+      - name: Creating alerts for canaries
```
Comment (brushknight) [inline .github/workflows/component_canary_alerts_delete.yml:59]: typo

File: .github/workflows/component_canary_alerts_delete.yml (hunk 1 above)
Comment (rubenruizdegauna) [reply .github/workflows/component_canary_alerts_delete.yml:1]: 🕵️ 👍

File: .github/workflows/component_canary_alerts_delete.yml (hunk 2 above)
Comment (rubenruizdegauna) [reply .github/workflows/component_canary_alerts_delete.yml:59]: 👍

**Review by brushknight — APPROVED** [review]
LGTM

**Pull request merged**